bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/dalitz_no_mll_cut/WH_012j_NLO_HtoElElGamma/WH_012j_NLO_HtoElElGamma_customizecards.dat

```diff
@@ -0,0 +1 @@
+set param_card mass 25 <MASS>
```

Comment:
You want to change the width accordingly or ignore as any small value is good enough?